src/test/scala-2.11/markets/participants/MarketParticipantSpec.scala

```diff
@@ -104,23 +90,25 @@ class MarketParticipantSpec extends TestKit(ActorSystem("MarketParticipantSpec")
 
   feature("A MarketParticipant actor should be able to add and remove markets.") {
 
-    val marketParticipant = TestActorRef(new TestMarketParticipant)
+    val markets = mutable.Map.empty[Tradable, ActorRef]
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1703602241/issues/source?bid=2921361&fileBranchId=3046267#l93)